.vscode/extensions.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
There aren't any conventions to my knowledge but I can tell you this from my experience:

Extension recommendations are intended to be used like I did here. [TypeScript](https://github.com/microsoft/TypeScript/blob/main/.vscode/extensions.json) does it like this too.

VSCode configurations like e.g. (build) tasks and debug configurations are sometimes shared like this. This is useful because debug commands for JS code can be very complex. (I haven't shared any yet because our commands are simple enough to fit inside npm scripts.)

VSCode settings like font size aren't typically shared for obvious reasons. 
However, some repos do include extension settings. Depends on the repo and organization.

I included extension settings because the two linter extensions can't do format on save by default. It's also not possible to enable format on save via a menu, so I just included the settings JSON to make it easier for newcomers.